app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+
+
+class InvalidCommandFormat(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) < 3 or parts[0] != "mv":
+        raise InvalidCommandFormat(
+            "Please provide a command like 'mv source_file destination_path'"
+        )
+
+    action = parts[0]
```

Comment:
There is no need for the `action` variable, remove it from here and if condition on line 20, the content of `parts[0]` was already checked in the first if condition.